includes/admin/import/class-batch-import-downloads.php

```diff
@@ -106,6 +106,21 @@ public function process_step() {
 					}
 				}
 
+				// Format the date properly
+				if ( ! empty( $args['post_date'] ) ) {
+
+					$timestamp = strtotime( $args['post_date'] );
```

Comment:
That would be correct. We want the timestamp to be consistent with the site's timezone. _Not_ setting it could give us incorrect results.